packages/react-server/package.json

```diff
@@ -33,14 +33,18 @@
     "@shopify/react-hydrate": "^1.1.4",
     "@shopify/react-network": "^3.3.1",
     "@shopify/sewing-kit-koa": "^6.1.2",
+    "@shopify/useful-types": "^1.3.0",
     "chalk": "^2.4.2",
     "koa": "^2.5.0",
     "koa-compose": "^4.1.0",
-    "koa-mount": "^4.0.0"
+    "koa-mount": "^4.0.0",
+    "fs-extra": "^8.1.0",
```

Comment:
What is this used for?

Reply:
It is used for checking if there is a Gemfile, when getting the manifest file.

```

function getManifestPath(root: string) {
  const gemFileExists = pathExistsSync(join(root, 'Gemfile'));
  if (!gemFileExists) {
    return;
  }

  // eslint-disable-next-line no-process-env
  return process.env.NODE_ENV === 'development'
    ? `tmp/sewing-kit/sewing-kit-manifest.json`
    : `public/bundles/sewing-kit-manifest.json`;
}
```

Reply:
Looks like it's an alias for the built in fs.existsSync.

https://github.com/jprichardson/node-fs-extra/blob/master/lib/path-exists/index.js#L11